### ISSUE TITLE
Ring / Notification to call somebody

### ISSUE THREAD

**Issue by biva** [mattermost/mattermost-plugin-jitsi#135]
We really love your plugin, thank you so much!

A great additional feature would be to ring the contacts I want to call within Mattermost: for example, add a specific notification with sound to the persons I mention after /jitsi. For example:

`/jitsi @tom` would trigger a sound/notification on Tom's app.

Would it be possible?

**Comment by jespino** [comment]
Hi @biva thanks for your feedback, really appreciated.

Technically is possible (not for mobile yet, but for the desktop/web app).

The proposed interface shouldn't work because `/jitsi @tom` would created a `@tom` named meeting room.

Maybe we can create a `/jitsi ring @tom` command to send the jitsi meeting link to tom in a direct message and start reproducing a "ring tone" there (Actually we can check if the user is online, before even trying that). For sure that command should be something that you can enable/disable from admin perspective and even from user perspective. I mean, as a I user, I may not want to receive this kind of "ring calls".

Anyway, I would love to have some input from @aaronrothschild perspective here :)

**Comment by biva** [comment]
Great, thanks a lot for your quick feedback, I totally agree with everything you write :) The single precision I'd like to make is that in our organisation, we disable every notification. So the Jitsi ring would be the single notification we accept. As a conclusion, we cannot rely on a notification that would be triggered only by a direct message, because we disabled notifications for direct messages. I hope I'm clear in my explanation, but I'm not sure :/

To summarize, the best would be that this ring is independent from other notifications set up in Mattermost.

**Comment by jespino** [comment]
Yes, i was thinking in a "ring tone" not a notification sound. Then you can have jitsi as a regular videoconference tool, and at the same time something like a "phone call".

**Comment by mhamiltonj** [comment]
This would be an excellent feature. Would be nice not to open a room and hope the person you're calling checks their messages.

**Comment by BananaAcid** [comment]
What is the current state? Anything I could help on testing?

**Comment by larkox** [comment]
@BananaAcid Right now the feature is open for contributions, but so far no one has taken it. If you are interested in developing it, you are more than welcome :smile:

**Comment by biva** [comment]
Hello, any news on this? We're forced to use Teams for now, and we'd love to use Jitsi to make calls with rings :)

**Comment by biva** [comment]
Hello @jespino @aaronrothschild Did you make any progress on this feature? Thanks :)

**Comment by aaronrothschild** [comment]
@biva there are no current plans to work on this feature, unfortunately.  While I don't like that you need to use MS teams for calling, as an FYI, there is also an MS Teams Plugin you can use with Mattermost to make meetings 1-click.

**Comment by BananaAcid** [comment]
We came from Skype, were checking out Mattermost and have the same problem as @biva

**Comment by ilolis** [comment]
Any news? This is a deal-breaker for many people who want to ditch Teams/Slack.
